builtin/credential/cert/path_login.go

```diff
@@ -646,6 +657,13 @@ func (b *backend) loadTrustedCerts(ctx context.Context, storage logical.Storage,
 			conf.QueryAllServers = conf.QueryAllServers || entry.OcspQueryAllServers
 		}
 	}
+
+	b.trustedCache.Add(certName, &trusted{
```

Comment:
nit: Should this only happen `if !b.trustedCacheDisabled`

Reply:
Definitely, otherwise we're just wasting memory.